modules/ROOT/partials/individually-licensed-components/export-to-pdf/export-to-pdf-api-usage.adoc

```diff
@@ -0,0 +1,41 @@
+[[api-usage]]
+== API Usage
+
+The {pluginname} On-Premises converter provides the ability to convert an HTML document to a PDF file via Restful API.
+
+The API is available on `http://localhost:[port]` (by default the `port` is `8080`).
+
+> The REST API documentation is available at `http://localhost:[port]/docs`.
```

Comment:
full URL not visible on the rendered page. should read `http://localhost:[port]/docs`.

Reply:
```suggestion
> The REST API documentation is available at `+http://localhost:[port]/docs+`.
```